client/extensions/woocommerce/app/order/order-refund-row.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * External dependencies
+ */
+import { localize } from 'i18n-calypso';
+import React, { Component, PropTypes } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import formatCurrency from 'lib/format-currency';
+
+class OrderRefundRow extends Component {
+	static propTypes = {
+		order: PropTypes.shape( {
+			currency: PropTypes.string.isRequired,
+			refunds: PropTypes.array.isRequired,
+		} ),
+	}
+
+	getRefundedTotal = ( order ) => {
+		return order.refunds.reduce( ( total, i ) => total + ( i.total * 1 ), 0 );
+	}
+
+	render() {
+		const { order, translate } = this.props;
+		const refundValue = order.refunds.length ? this.getRefundedTotal( order ) : false;
+		if ( ! refundValue ) {
+			return null;
+		}
+
+		return (
+			<div className="order__details-total-refund">
+				<div className="order__details-totals-label">{ translate( 'Refunded' ) }</div>
```

Comment:
Hi! I've found a possible matching string that has already been translated **37** times:
`translate( 'Refunded To' )` *ES Score: 12*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/fix/store-16645-add-tax-to-refunds)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).